# request fields
nav_msgs/OccupancyGrid map
---
# response fields
bool success

